CentralDeErros.Data/Maps/OccurrenceMap.cs

```diff
@@ -41,6 +39,24 @@ public void Configure(EntityTypeBuilder<Occurrence> builder)
                 .Property(k => k.OccurrenceDate)
                 .HasColumnName("occurrence_date")
                 .IsRequired();
+
+            builder
+                .HasOne(k => k.Microsservice)
+                .WithMany(s => s.Occurrences)
+                .HasForeignKey(x => x.MicrosserviceId)
```

Comment:
Sobre o HasForeignKey, vamos dar uma pesquisada se a declaração precisa ser explícita, por que acho que o o HasOne + HasMany já gera a FK no EF Core (junto com os atributos de navegação na classe da Entidade).

Reply:
Tinha testado sem o HasForeingKey no Insomnia e funcionou apenas com essas outras marcações que listei no primeiro comentário.

Reply:
Vdd, estou pesquisando aqui para entender se precisa ficar explicito ou implícito, incluir assim pois como tinha que inserir as demais acabei alterando para ficar padrão. 

Nós exemplos que vi estavam com a declaração da FK. 

Confesso que para mim o que deve ficar explicito e implícito ainda é uma linha muito tênue mas com o tempo vamos pegando.

Reply:
Cara aqui acabei colocando um Is.Requered, mas acho que deve ser retirada, você concorda? 

Será que precisamos declarar o nome da coluna?

Reply:
Pelo que entendi, @MarceloAmorim25 , o HasForeignKey funciona pra setar numa propriedade já criada na parte do C# mesmo